### PR TITLE
SAK-40577 - Update Spring to latest release before 5.X

### PR DESCRIPTION
**Pull request by deinok**
This update will help to track deprecations done before 5.X making more easy to port all sakai to 5.X

### CODE REVIEW

**Comment by ern** [comment]
@deinok Open a new JIRA issue that says:
**Spring upgrade from 4.x.x. to 4.x.x**
put all the commits required for this change and submit a single PR

Please stop using SAK-40004

**Comment by deinok** [comment]
It's related to the port to JDK11

File: master/pom.xml [hunk 1]
```diff
@@ -57,13 +57,7 @@
     <sakai.simple-xml.version>2.6.4</sakai.simple-xml.version>
     <sakai.spring.groupId>org.springframework</sakai.spring.groupId>
     <sakai.spring.artifactId>spring-core</sakai.spring.artifactId>
-    <sakai.spring.version>4.3.14.RELEASE</sakai.spring.version>
```
Comment (ern) [inline master/pom.xml]: These properties exist to make it easier to set specific versions without having to adjust the the actual dependencies constantly. Please leave these in place and use them accordingly.

**Comment by ern** [comment]
Core Team decided to close.

**Pull request closed without merging**